docker-compose.override.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+    assets-builder:
+        volumes:
+            - ./:/srv/pattern-library
+            - /srv/pattern-library/node_modules
+            - source_assets:/srv/pattern-library/source/assets
```

Comment:
We're not expecting these to go anywhere yet, just demonstrating that gulp watch works?